core-v1/src/main/scala/lens/VariantLens.scala

```diff
@@ -144,7 +146,19 @@ object VariantLens extends VariantLensLowPriorityImplicits {
 sealed trait VariantLensLowPriorityImplicits {
 
   /**
-    * Wrap the result of this lens with an [[AsIterableBuilder]] for helper operations on [[IterableOnce]] types.
+    * Wrap the result of this lens with an [[VariantLens.AsIterableBuilder]] for helper operations
+    * on [[IterableOnce]] types.
+    */
+  implicit def asFoldableIterable[A, B, C[_] : Foldable, E](
+    lens: VariantLens[A, B],
+  )(implicit
+    ev: B <:< C[E],
+  ): VariantLens.AsIterableBuilder[A, E] =
+    new VariantLens.AsIterableBuilder(lens.asIterable)
```

Comment:
This seems to fail when you also have `import cats._` because it will assume the `Foldable[Id]` for collection types... I wonder if there is a way to fix this.